src/js/app.js

```
@@ -1271,8 +1271,8 @@ function renderIssueCard(iss){
   const gfiNames=['good first issue','good-first-issue'];
   const otherLabels=iss.labels.filter(l=>!gfiNames.includes(String(l).toLowerCase())).slice(0,2)
     .map(l=>`<span class="issue-label" style="background:rgba(107,33,168,.06);color:var(--purple);border:1px solid rgba(107,33,168,.2)">${escapeHtml(l)}</span>`).join('');
-  const safeHref = validateIdeasUrl(iss.url);
-  const imgSrc = validateIdeasUrl(iss.logo);
+  const safeHref = validateUrl(iss.url);
```

ESLint

[error] 'validateUrl' is not defined.

(no-undef)